tests/test_encode_decode.py

```diff
@@ -254,9 +255,32 @@ def test_rgb_hsv_round_trip(self, r, g, b):
         self.assertColorsValid(h=(h, h2), s=(s, s2), v=(v, v2))
 
 
+text_strategy = st.text(alphabet=st.characters(blacklist_categories=["Cc", "Cs"]))
+
+plistlib_data = st.recursive(
+    st.booleans()
+    | st.integers(min_value=-1 << 63, max_value=1 << 64 - 1)
+    | st.floats(allow_nan=False)
+    | text_strategy,
+    lambda sub_strategy: st.lists(sub_strategy)
+    | st.dictionaries(text_strategy, sub_strategy),
+)
```

Comment:
> Values can be strings, integers, floats, booleans, tuples, lists, dictionaries (but only with string keys), Data, bytes, bytesarray or datetime.datetime objects.

Let's at least add `st.binary()`, `st.datetimes()`, and `st.lists(sub_strategy).map(tuple)` to the strategy.

Reply:
I tried adding `st.lists(sub_strategy).map(tuple)`. `plistlib` treats `tuple` and `list` as interchangeable. So a dumped `tuple`, will get loaded back as `list`. Is this a bug in the library?
```
Falsifying example: test_dumps_loads(
    self=<test_encode_decode.TestPlistlib testMethod=test_dumps_loads>,
    payload=(),
    fmt=<PlistFormat.FMT_XML: 1>,
    pass_format_arg=False,
)


[] != ()

Expected :()
Actual   :[]
```

Reply:
Probably a case for better documentation - you could write it!  e.g. integers only if signed-64bit, datetimes rounded down to the minute, tuples accepted but stored as lists.